3rdparty/kuserfeedback/CMakeLists.txt

```diff
@@ -24,6 +23,11 @@ list(APPEND gammaray_userfeedback_srcs
     core/auditloguicontroller.cpp
     widgets/auditlogbrowserdialog.cpp
 )
+if (Qt5OpenGL_FOUND)
+    list(APPEND gammaray_userfeedback_srcs
+        core/openglinfosource.cpp
+    )
+endif()
```

Comment:
Don't remove openglinfosource.cpp completely. The cpp file already contains an #if to check for Qt >= 5.0.0, reuse that to also check for `!defined(QT_NO_OPENGL)`

Reply:
@akreuzkamp Thanks for the feedback. Just fixed the code ;)!